recipes/openblas/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+from conans import ConanFile, CMake, tools
+import sys
+import os
+
+
+class OpenBLAS(ConanFile):
+    name = "openblas"
+    version = "0.3.7"
+    license = "BSD 3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.openblas.net"
+    description = "An optimized BLAS library based on GotoBLAS2 1.13 BSD version"
+    topics = (
+        "OpenBLAS",
+        "BLAS",
+        "LAPACK"
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_lapack": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "build_lapack": False
+    }
+    generators = "cmake", "cmake_find_package"
+    no_copy_source = True
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename('OpenBLAS-{}'.format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        if self.options.build_lapack:
+            self.output.warn(
+                "Building with lapack support requires a Fortran compiler.")
+
+        cmake.definitions["NOFORTRAN"] = not self.options.build_lapack
+        cmake.definitions["BUILD_WITHOUT_LAPACK"] = not self.options.build_lapack
+
+        if self.settings.compiler == "Visual Studio" and not self.options.shared:
+            cmake.definitions["MSVC_STATIC_CRT"] = True
+
+        cmake.configure(
+            build_folder=self._build_subfolder,
+            source_folder=self._source_subfolder)
```

Comment:
You should use Conan CMake Wrapper so insert all CMake settings. Using _source_subfolder directly, without conan_basic_setup() can result in an incorrect package.

Reply:
I can't seem to get it to work using a wrapper. I have pushed what I  have, but it seems it is not copying over the CMakeLists.txt. Any suggestions on how to make it work?

Reply:
@uilianries friendly ping :)

Reply:
You don't seem to have actually added a CMake wrapper (`CMakeLists.txt` file at the root level of the recipe)

You should be able to copy https://github.com/bincrafters/templates/blob/master/default/CMakeLists.txt

Reply:
@bmanga please follow Croydon instructions. You need a cmake wrapper to fix your build,

Reply:
My bad, I had placed the CMakeLists in the recipe directory, and forgot to commit that. Sorry about that :(